ios/eMobility.xcodeproj/project.pbxproj

```diff
@@ -967,7 +967,6 @@
 					"\"${PODS_CONFIGURATION_BUILD_DIR}/React-RCTAnimation\"",
 					"\"${PODS_CONFIGURATION_BUILD_DIR}/React-RCTBlob\"",
 					"\"${PODS_CONFIGURATION_BUILD_DIR}/React-RCTImage\"",
-					"\"${PODS_CONFIGURATION_BUILD_DIR}/React-RCTLinking\"",
```

Comment:
Are you sure about that one? It's a react-native default pod that should be required at build.

Reply:
No I'm not, I dont understand why this change appeared